.github/workflows/pong-server.yaml

```diff
@@ -60,6 +60,7 @@ jobs:
                   export PONG_GAME_SERVERS_MAX_PORT=42210
                   export GAME_SERVER_PATH=`pwd`/pong_server/src/game_server/
                   export PATH_TO_SSL_CERTS=`pwd`/ssl/certs/
+                  export DEBUG=True
```

Comment:
Why is the workflow in debug mode?

Reply:
Because I can't access the redis cache without starting the docker so I use a different cache system for the unit tests